config/locales/en.yml

```diff
@@ -1656,7 +1656,7 @@ en:
     errors:
       code_401: "You need to be authenticated to access this resource."
       code_403: "You are not authorized to access this resource."
-      code_404: "Couldn\'t find %{type} with id=%{id}"
```

Comment:
@Azure7111 I might have missed part of specification discussion. Is making this generic really desirable?